content/pl/docs/tutorials/_index.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Samouczki
+main_menu: true
+weight: 60
+content_template: templates/concept
+---
+
+{{% capture overview %}}
+
+W tym rozdziale dokumentacji Kubernetes znajdziesz różne samouczki.
+Dzięki nim dowiesz się, jak osiągnąć złożone cele, które przekraczają wielkość
+pojedynczego [zadania](/docs/tasks/). Typowy samouczek podzielony jest na kilka części,
+z których każda zawiera sekwencję odpowiednich kroków.
+Przed zapoznaniem się z samouczkami warto stworzyć zakładkę do
+[słownika](/docs/reference/glossary/), aby móc się później do niego na bieżąco odwoływać.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Podstawy
+
+* [Podstawy Kubernetes](/docs/tutorials/kubernetes-basics/) to interaktywny samouczek, który pomoże zrozumieć system Kubernetes i wypróbować jego podstawowe możliwości.
+
+* [Scalable Microservices with Kubernetes (Udacity)](https://www.udacity.com/course/scalable-microservices-with-kubernetes--ud615)
+
+* [Introduction to Kubernetes (edX)](https://www.edx.org/course/introduction-kubernetes-linuxfoundationx-lfs158x#)
+
+* [Hello Minikube](/docs/tutorials/hello-minikube/)
```

Comment:
Should we leave `Hello Minikube` or go hard and translate it to `Witaj Minikube`?

Reply:
Most localizations leave it as Hello Minikube. As Polish uses the same alphabet as English, I would leave it.